cinema/models.py

```diff
@@ -35,12 +40,21 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_path(movie: "Movie", filename: str) -> str:
+    _, ext = os.path.splitext(filename)
+    return os.path.join(
+        "uploads/movies/",
+        f"{slugify(movie.title)}-{uuid.uuid4()}{ext}"
+    )
```

Comment:
```suggestion
def movie_image_path(movie: "Movie", filename: str) -> str:
    _, ext = os.path.splitext(filename)
    return os.path.join(
        "uploads", "movies",
        f"{slugify(movie.title)}-{uuid.uuid4()}{ext}"
    )
```

Read how this method works